.github/workflows/deploy-to-uat.yml

```diff
@@ -9,7 +9,7 @@ env:
 
 jobs:
   build-uat:
-    if: github.actor == 'JamesChristianson' || github.actor == 'emailtokvk' || github.actor == 'thomaslkp123' || github.actor == 'rajchala' || github.actor == 'TasTasniem'
+    if: github.actor == 'JamesChristianson' || github.actor == 'emailtokvk' || github.actor == 'thomaslkp123' || github.actor == 'rajchala' || github.actor == 'TasTasniem' || github.actor == 'kelechi-i'
```

Comment:
Why does kelechi-i need to be able to push to UAT?

Reply:
Same rights as tester he is covering for - anyway this is in main - so reason it is here is probable because main has been pulled into this branch. If we need to revoke this access we should do it in main.